app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="by.paranoidandroid.dailyvisualizer">
-
+    <uses-feature android:name="android.hardware.camera"  android:required="true" />
```

Comment:
Why is camera feature required? Maybe it's better to replace it with false, isn't it?
`<uses-feature android:name="android.hardware.camera"  android:required="false" />  `

Reply:
I agree with you)